qa-tests-backend/src/main/groovy/services/SecretService.groovy

```diff
@@ -22,7 +26,7 @@ class SecretService extends BaseService {
 
     static waitForSecret(String id, int timeoutSeconds = 10) {
         int intervalSeconds = 1
-        int retries = timeoutSeconds / intervalSeconds
+        int retries = (timeoutSeconds / intervalSeconds) as int
```

Comment:
Nit: Is this just a fancy way to say `int retries = 10`? Why are we doing this?

Reply:
`timeoutSeconds` is a param with default value if not passed then 10 will be used.
But we always pass 10. This service looks weird. And I don't like retruning null instead of passing the exception. Probably could be refactored with `@Retry` from:
- https://github.com/stackrox/stackrox/pull/14624